opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -197,27 +197,32 @@ def schema_url(self):
         """
         return self._schema_url
 
-    def _check_instrument_id(
+    def _is_instrument_registered(
         self, name: str, type_: type, unit: str, description: str
-    ) -> bool:
+    ) -> Tuple[bool, str]:
         """
         Check if an instrument with the same name, type, unit and description
         has been registered already.
-        """
 
-        result = False
+        Returns a tuple. The first value is `True` if the instrument has been
+        registered already, `False` otherwise. The second value is the
+        instrument id.
+        """
 
         instrument_id = ",".join(
             [name.strip().lower(), type_.__name__, unit, description]
         )
 
+        result = None
```

Comment:
Do we need the differetiation between none and false here? if not, why not initialize to False

Reply:
Fixed